corehq/apps/export/export.py

```diff
@@ -465,25 +465,29 @@ def _get_base_query(export_instance):
         )
 
 
-def rebuild_export(export_instance, last_access_cutoff=None, filters=None):
+def rebuild_export(export_instance, filters=None):
     """
     Rebuild the given daily saved ExportInstance
     """
-    if _should_not_rebuild_export(export_instance, last_access_cutoff):
-        return
     filters = filters or export_instance.get_filters()
     export_file = get_export_file([export_instance], filters or [])
     with export_file as payload:
         save_export_payload(export_instance, payload)
 
 
-def _should_not_rebuild_export(export, last_access_cutoff):
+def _should_rebuild_export(export, last_access_cutoff=None):
+    """
+    :param last_access_cutoff: Any exports not accessed since this date will not be rebuilt.
+    :return: True if export should be rebuilt
+    """
     # Don't rebuild exports that haven't been accessed since last_access_cutoff or aren't enabled
-    return (
-        last_access_cutoff
-        and export.auto_rebuild_enabled
-        and export.last_accessed
-        and export.last_accessed < last_access_cutoff
+    is_auto_rebuild = last_access_cutoff is not None
+    return not is_auto_rebuild or (
```

Comment:
The above two lines are a confusing. It seems to be saying "rebuild if not auto rebuild," which seems like a contradiction. Maybe I'm not understanding the context? Is it true that a rebuild should always happen if `last_access_cutoff is None`? If so, maybe simplify this to
```py
return last_access_cutoff is None or (
```

Reply:
yes, that is true. I introduced the variable to make it clearer what `last_access_cutoff is None` means. `last_access_cutoff` is only `None` if the function is called directly from a user request as opposed to via the scheduled task.

Fine to change it back, just thought it made this easier to read.